.github/src/context.js

```diff
@@ -96,7 +96,7 @@ export async function extractInputs(github, context, core) {
         context.payload
       );
 
-    let issue_number;
+    let issue_number = NaN;
```

Comment:
```suggestion
    let issue_number;
```

Better to require each "if" block to set this explicitly to prevent errors.